jmix-rest/rest/src/main/java/io/jmix/rest/impl/service/EntitiesControllerManager.java

```diff
@@ -287,20 +282,24 @@ public Long countSearchEntities(String entityName, String searchRequestBody) {
         return countSearchEntities(entityName, searchEntitiesRequest.getFilter().toString(), searchEntitiesRequest.getModelVersion());
     }
 
-    protected String _loadEntitiesList(String queryString,
+    protected String _loadEntitiesList(Condition condition,
```

Comment:
Do not use "_" in names. If you cannot invent a new name, use "Internal" suffix, like `loadEntitiesListInternal`.
But here `loadEntitiesJson()` would be a good name.